package.json

```diff
@@ -9,78 +9,78 @@
     "bunyan": "^1.8.15",
     "bunyan-format": "^0.2.1",
     "connect-flash": "^0.1.1",
     "connect-redis": "^6.1.3",
     "cookie-parser": "^1.4.6",
     "cookie-session": "^2.0.0",
     "csurf": "^1.11.0",
-    "csv-parse": "^5.0.4",
-    "dotenv": "^16.0.0",
+    "csv-parse": "^5.1.0",
+    "dotenv": "^16.0.1",
     "escape-html": "^1.0.3",
-    "express": "^4.17.3",
-    "express-form-data": "^2.0.17",
-    "express-session": "^1.17.2",
-    "googleapis": "^100.0.0",
+    "express": "^4.18.1",
+    "express-form-data": "^2.0.18",
+    "express-session": "^1.17.3",
+    "googleapis": "^101.0.0",
     "govuk-frontend": "^3.14.0",
-    "helmet": "^5.0.2",
+    "helmet": "^5.1.0",
     "http-errors": "^2.0.0",
     "jsonwebtoken": "^8.5.1",
     "jwt-decode": "^3.1.2",
-    "moment": "^2.29.2",
+    "moment": "^2.29.3",
     "moment-business-days": "^1.2.0",
-    "nocache": "^3.0.3",
+    "nocache": "^3.0.4",
     "notifications-node-client": "^5.1.1",
     "nunjucks": "^3.2.3",
-    "passport": "^0.5.2",
+    "passport": "^0.6.0",
     "passport-oauth2": "^1.6.1",
-    "qs": "^6.10.3",
+    "qs": "^6.10.5",
     "redis": "^3.1.2",
-    "superagent": "^7.1.1"
+    "superagent": "^7.1.6"
   },
   "devDependencies": {
     "@types/axios": "^0.14.0",
     "@types/bunyan": "^1.8.8",
-    "@types/cookie-parser": "^1.4.2",
+    "@types/cookie-parser": "^1.4.3",
     "@types/express": "^4.17.13",
     "@types/express-session": "^1.17.4",
     "@types/jest": "^27.4.1",
     "@types/node": "^16.11.10",
     "@types/superagent": "^4.1.15",
-    "@typescript-eslint/eslint-plugin": "^5.19.0",
-    "@typescript-eslint/parser": "^5.19.0",
-    "audit-ci": "^6.2.0",
-    "concurrently": "^7.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.27.1",
+    "@typescript-eslint/parser": "^5.27.1",
+    "audit-ci": "^6.3.0",
+    "concurrently": "^7.2.1",
     "cypress": "^9.5.4",
     "cypress-multi-reporters": "^1.5.0",
     "dotenv-expand": "^8.0.3",
-    "eslint": "^8.13.0",
+    "eslint": "^8.17.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-import-resolver-typescript": "^2.7.1",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-no-only-tests": "^2.6.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "husky": "^7.0.4",
+    "husky": "^8.0.1",
     "jest": "^27.5.1",
     "jest-html-reporter": "^3.4.2",
     "jest-junit": "^13.1.0",
-    "lint-staged": "^12.3.7",
+    "lint-staged": "^13.0.1",
     "mocha-junit-reporter": "^2.0.2",
-    "nock": "^13.2.4",
-    "nodemon": "^2.0.15",
+    "nock": "^13.2.6",
+    "nodemon": "^2.0.16",
     "prettier": "^2.6.2",
-    "sass": "^1.50.0",
-    "set-cookie-parser": "^2.4.8",
-    "supertest": "^6.2.2",
+    "sass": "^1.52.3",
+    "set-cookie-parser": "^2.5.0",
+    "supertest": "^6.2.3",
     "ts-jest": "^27.1.4",
-    "typescript": "^4.6.3"
+    "typescript": "^4.7.3"
   },
   "scripts": {
```

Comment:
this still needs the following updates, but they're quite large changes so spinning off into separate PRs: 

* redis
* cypress
* govuk frontend 

Also jest but that requires a defect in jest-html-reporter to be fixed first